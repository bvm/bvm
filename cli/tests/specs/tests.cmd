@echo off

CALL bvm install --use ../../../target/debug/args_test_util.json
CALL args_test_util "console.log(\"hello\")"
CALL args_test_util "console.log(2 != 3)"
REM cmd supports no quotes here, but powershell and cmd both require them
CALL args_test_util JSON.stringify({})
CALL args_test_util "JSON.stringify({})"
CALL args_test_util lib=""
CALL args_test_util lib=test,other
