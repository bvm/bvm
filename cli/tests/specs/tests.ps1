& bvm install --use target/debug/args_test_util.json
# for some reason, double quotes within double quotes escaped with backticks wasn't working even with the exe
# & args_test_util 'console.log(`"hello`")'
& args_test_util "console.log(2 != 3)"
& args_test_util "JSON.stringify({})"
& args_test_util lib=""
& args_test_util lib=test,other
