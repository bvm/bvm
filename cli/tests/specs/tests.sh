#!/bin/sh

& bvm install --use ../../target/debug/args_test_util.json
args_test_util "console.log(\"hello\")"
args_test_util "console.log(2 != 3)"
args_test_util "JSON.stringify({})"
args_test_util lib=""
args_test_util lib=test,other