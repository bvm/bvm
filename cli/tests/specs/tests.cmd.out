["console.log(\"hello\")"]
["console.log(2 != 3)"]
["JSON.stringify({})"]
["JSON.stringify({})"]
["lib="]
["lib=test,other"]
